src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -43,7 +43,7 @@
 
   private static final int NUMBER_OF_PRIVATE_MESSAGES = 10;
 
-  private ChatPageObject openChatForUser(
+  private ChatPageObject OPENCHATFORUSERS(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)